region/norfair/west/Crumble Shaft.json

```diff
@@ -406,6 +425,50 @@
         "Climb up 3 screens, and perform a turnaround buffered spin-jump away from the door to trigger the transition, bypassing any lock on the door."
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "G-Mode Setup - Get Hit By Sova",
+      "requires": [
+        {"heatFrames": 4000},
+        {"or": [
+          "SpaceJump",
+          "h_canUseSpringBall",
+          {"and": [
+            "canStaggeredWalljump",
```

Comment:
The easiest way I saw to do this was to follow behind the Sova and pass it when you are near the top by jumping off the side of the top-middle crumble blocks.  I don't really see how the extra requirements are meant to be used.
And my room times were 36 seconds for staggered wall jump, 36 for spacejump and 35 for springball.

Reply:
I was climbing the left wall. this does seem much better